src/navigation/DevPersistedNavigationContainer.tsx

```diff
@@ -63,8 +63,8 @@ function DevPersistedNavigationContainerImpl(
 
   return (
     <NavigationContainer
-      {...others}
       key={persistKey}
+      {...others}
```

Comment:
What is the purpose of this change?

Reply:
Updated linter says key prop needs to come before spread